cmd/cstorvolumeclaim/cstorvolumeclaim.go

```diff
@@ -447,11 +449,24 @@ func getUniquePoolList(pvName string, poolList *apis.CStorPoolList) *apis.CStorP
 		return nil
 	}
 
-	usedPoolMap := getUniqueUsedPoolNames(cvrList)
+	usedPoolMap := getUsedPoolNames(cvrList)
 	for _, pool := range poolList.Items {
 		if !usedPoolMap[pool.Name] {
-			uniquePool.Items = append(uniquePool.Items, pool)
+			usablePoolList.Items = append(usablePoolList.Items, pool)
 		}
 	}
-	return uniquePool
+	return randomizePoolList(usablePoolList)
+}
+
+// randomizePoolList returns randomized pool list
+func randomizePoolList(list *apis.CStorPoolList) *apis.CStorPoolList {
+	res := &apis.CStorPoolList{}
+
+	r := rand.New(rand.NewSource(time.Now().Unix()))
+	perm := r.Perm(len(list.Items))
+	for _, randomIdx := range perm {
+		res.Items = append(res.Items, list.Items[randomIdx])
+	}
+
```

Comment:
@AmitKumarDas play.golang.org caches the result of runs, so if you type the same
program twice there is a possibility that the response will be fetched
from cache instead of being compiled and ran

So even if you changed the Seed you could observe the same results
(when running on the play.golang.org).

To make sure that your are with a fresh result, change something
before running or run the code on your computer.

Reply:
do we have the UT that confirms this randomization.